components/signup/signup_controller/signup_controller.jsx

```diff
@@ -39,10 +41,13 @@ export default class SignupController extends React.PureComponent {
         ldapLoginFieldName: PropTypes.string.isRequired,
         openidButtonText: PropTypes.string,
         openidButtonColor: PropTypes.string,
+        subscriptionStats: PropTypes.object,
```

Comment:
We can use the [`SubscriptionsStats` type](https://github.com/mattermost/mattermost-redux/blob/4142a2a0a19f414e2d2cd8d4d3da7a206cd38efe/src/types/cloud.ts#L112-L114) here instead of object

Reply:
I think there is a limitation to do this. Since this file is still .jsx and uses proptypes, and the desired type is typescript. Eventually when this file gets migrated to .tsx, we will definitely use those types.

Reply:
Ah yes, completely missed that it was not migrated to typescript yet.